modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-fido2-web/src/main/resources/META-INF/resources/js/RegistrationTransformer.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import * as webauthn from './webauthn';
+
+export default function RegistrationTransformer({
+	additionalProps,
+	portletNamespace,
+	...otherProps
+}) {
+	return {
+		...otherProps,
+		onClick() {
+			const {pkccOptions} = additionalProps;
+
+			if (!pkccOptions) {
+				return;
+			}
+
+			webauthn
+				.create(JSON.parse(pkccOptions))
+				.then((value) => {
+					const responseInput = document.getElementById(
+						`${portletNamespace}responseJSON`
+					);
+
+					const publicKeyCredentials = webauthn.responseToObject(
```

Comment:
I tried to run registration, but it failed at this line as **webauthn.responseToObject()** is undefined.
I looked into **webauthn.js** and noticed the method is now called **credentialToObject()**, so I modified it to **webauthn.credentialToObject()**, however it still fails because of undefined, not sure what's problem? 
In browser console, however, I can access **webauthn.credentialToObject()** though.

Reply:
@arthurchan35 you're right, I forgot to change that